R/utils_render_html.R

```diff
@@ -108,7 +112,7 @@ get_table_defs <- function(data) {
     table_style <- paste(table_style, paste0("width: ", table_width), sep = "; ")
   }
 
-  # Create the <colgroup> tag
+  # Create the `<colgroup>` tag
   table_colgroups <-
     htmltools::tags$colgroup(
       lapply(widths, function(width) {
```

Comment:
Use `css()` so that width will be dropped if it's null or empty string

Reply:
Made this change. Thanks for the suggestion.